include/core/lethe_grid_tools.h

```diff
@@ -330,7 +330,7 @@ namespace LetheGridTools
    * @param[in] line_start The starting point of the line
    * @param[in] line_direction The direction of the line
    * @param[in] sphere_center The center of the sphere
-   * @param[in] sphere_radius The radius of the sphere
+   * @param[in] sphere_diameter The diameter of the sphere
```

Comment:
Is there a reason for changing the function instead of just passing diameter *0.5? Did you check that this function is not anywhere else?

Reply:
I double check, it is not used anywhere else.

Reply:
Yes and no, because you are multiplying by 0.5 within the function itself with the local copy.
You could have just kept the function as-is and called it with 0.5*particle_diameter as an argument. It would not have made a difference.
The function is not used elsewhere, so it'S not a big deal, but if it had been, it could have broken the entire code. Makes me think that we should really write a unit test for this function...